package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "nib",
   "description": "Stylus mixins and utilities",
-  "version": "1.1.3",
```

Comment:
bumping the version should be done in a separate tagged commit

Reply:
@notslang resolved